scrapers_next/in/committees.py

```diff
@@ -0,0 +1,64 @@
+from spatula import URL, HtmlListPage, SkipItem
+
+from openstates.models import ScrapeCommittee
+
+
+class CommitteeDetail(HtmlListPage):
+    source = URL(
+        "https://tlhgp53g3c.execute-api.us-east-2.amazonaws.com/beta/api/getMembers?committee_id=840b8a47-2a40-4e28-9e2a-68e6c66c9e45&session_lpid=session_2021"
```

Comment:
We'll need a way to dynamically update the session. Also, clarifying how you found this AWS endpoint that has IN API calls would be helpful.

Reply:
from this site: 
https://beta.iga.in.gov/2022/committees
currently it works without session.

Reply:
Added the header currently website not sending any session to API... when they do then we will update.
This API is from their new site as their old website https://iga.in.gov/ will not be available after the middle of CY 2023.